torch/jit/_isinstance.py

```diff
@@ -0,0 +1,109 @@
+from typing import List, Dict, Tuple, Union, Optional
+
+
+def get_origin(the_type):
+    return getattr(the_type, "__origin__", None)
+
+
+def get_args(the_type):
+    return getattr(the_type, "__args__", None)
+
+
+def check_args_exist(the_type):
```

Comment:
yes, can do

Reply:
So actually, looking more into this, the is_list/is_dict/etc in _jit_internal.py do more checks that what is necessary for the actual torch.jit.isinstance needs. The first check in the _jit_internal.py which triggers the shared error message is the only real things that _isintance needs, the return values from the _jit_internal.py aren't needed. I see the benefit of using the _jit_internal.py functions because the error messages should be the same and it is code duplication to have them twice. I can see a couple options:

1. use the _jit_internal.py functions in _isinstance and perform the unnecessary checks 
2. re-write the error messages in a new function which both the _jit_intenral.py functions and  _isinstnace functions can utilize
3. keep the code as is in this PR